### PR TITLE
Fixed vLLM documentation

### PR DESCRIPTION
**Pull request by varunshenoy**
Updated predict dictionary to include model name --published`.

### CODE REVIEW

File: pyproject.toml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 [tool.poetry]
 name = "truss"
-version = "0.6.0"
+version = "0.5.7rc1"
```
Comment (bolasim) [inline pyproject.toml:3]: this should be bumped to 0.6.1

File: pyproject.toml [hunk 2]
```diff
@@ -41,6 +41,7 @@ watchfiles = "^0.19.0"
 huggingface_hub = "^0.16.4"
 rich-click = "^1.6.1"
 inquirerpy = "^0.3.4"
+google-cloud-storage = "^2.10.0"
```
Comment (bolasim) [inline pyproject.toml:44]: is this needed here or just in builder?

File: pyproject.toml [hunk 3]
```diff
@@ -1,6 +1,6 @@
 [tool.poetry]
```
Comment (bolasim) [inline pyproject.toml:1]: maybe revert this whole file for this PR?

**Review by bolasim — APPROVED** [review]
please revert pyproject.toml before merging

File: pyproject.toml (hunk 3 above)
Comment (varunshenoy) [reply pyproject.toml:1]: good call -- do you know why the tests here are failing though? Will be important for the gcs pr.

**Pull request merged**